v1/content.yaml

```diff
@@ -518,104 +530,114 @@ paths:
-      x-request-handler:
-        - get_from_backend:
-            request:
-              uri: /{domain}/sys/parsoid/data-parsoid/{title}
-              headers:
-                cache-control: '{cache-control}'
-                if-unmodified-since: '{if-unmodified-since}'
-                x-restbase-mode: '{x-restbase-mode}'
-                x-restbase-parentrevision: '{x-restbase-parentrevision}'
-      x-monitor: true
-      x-amples:
-        - title: Get data-parsoid by title
-          request:
-            params:
-              domain: en.wikipedia.org
-              title: Foobar
-          response:
-            status: 200
-            headers:
-              etag: /.+/
-              content-type: application/json
-            body:
-              counter: /\d+/
-              ids: /.*/
-              sectionOffsets: /.*/
+#  /{module:page}/data-parsoid/{title}:
+#    get:
+#      tags:
+#        - Page content
+#      summary: Get latest data-parsoid metadata for a title.
+#      description: |
+#        Data-parsoid is metadata used by
+#        [Parsoid](https://www.mediawiki.org/wiki/Parsoid) to support clean
+#        round-tripping conversions between HTML and Wikitext. Among other
+#        things, it contains the original Wikitext offsets of each HTML
+#        element, keyed by element ID. The format is unstable.
+#
+#        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Unstable)
+#      parameters:
+#        - name: title
+#          in: path
+#          description: the title of page content
+#          type: string
+#          required: true
+#      produces:
+#        - application/json
+#      responses:
+#        '200':
+#          description: data-parsoid JSON blob
+#          schema:
+#            $ref: '#/definitions/data-parsoid'
+#        '404':
+#          description: Unknown page title
+#          schema:
+#            $ref: '#/definitions/problem'
+#        default:
+#          description: Error
+#          schema:
+#            $ref: '#/definitions/problem'
+#      x-request-handler:
+#        - get_from_backend:
+#            request:
+#              uri: /{domain}/sys/parsoid/data-parsoid/{title}
+#              headers:
+#                cache-control: '{cache-control}'
+#                if-unmodified-since: '{if-unmodified-since}'
+#                x-restbase-mode: '{x-restbase-mode}'
+#                x-restbase-parentrevision: '{x-restbase-parentrevision}'
+#      x-monitor: true
+#      x-amples:
+#        - title: Get data-parsoid by title
+#          request:
+#            params:
+#              domain: en.wikipedia.org
+#              title: Foobar
+#          response:
+#            status: 200
+#            headers:
+#              etag: /.+/
+#              content-type: application/json
+#            body:
+#              counter: /\d+/
+#              ids: /.*/
+#              sectionOffsets: /.*/
 
   /{module:page}/data-parsoid/{title}/:
```

Comment:
Since we store `html` and `data-parsoid` together, this endpoint will always have the same result as `html` listing. So may be we could remove it too?